jetty-core/jetty-xml/src/main/java/org/eclipse/jetty/xml/XmlConfiguration.java

```diff
@@ -735,12 +733,8 @@ private void set(Object obj, XmlParser.Node node) throws Exception
             String message = oClass + "." + setter + "(" + vClass[0] + ")";
             if (types != null)
                 message += ". Found setters for " + types;
-            NoSuchMethodException failure = new NoSuchMethodException(message);
-            for (int i = 0; i < me.size(); i++)
-            {
-                failure.addSuppressed(me.getThrowable(i));
-            }
-            throw failure;
+
+            throw TypeUtil.withSuppressed(new NoSuchMethodException(message), errors);
```

Comment:
There's no test for `errors.isEmpty()` here as at this point we must throw an exception.